_data/utilities.yml

```
@@ -184,15 +185,15 @@ websites:
     tfa:
       - sms
       - phone
-      - software
+      - totp
```

They use 2FA via push notifications on some androids as well.
```suggestion
      - totp
      - proprietary
```

---

Other than that everything looks good :+1: